### PR TITLE
Feat: sponsored banners

### PR DESCRIPTION
**Pull request by luarakerlen**
#### What is the purpose of this pull request?

We need to create Sponsored Banners for our clients. To do this, it's necessary to request the banners data to Ad Server.
This PR creates the request to Ad Server and exports the resolver for Sponsored Banners component can use it.

#### Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (changes configuration files, GitHub workflows, etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/clients/AdServer.ts [hunk 1]
```diff
@@ -27,6 +32,20 @@ class AdServer extends ExternalClient {
       ...body,
     })
   }
+
+  public getSponsoredBanners(
+    body: AdServerSponsoredBannersRequest
+  ): Promise<AdServerSponsoredBannersResponse> {
+    if (process.env.VTEX_APP_LINK) {
+      // eslint-disable-next-line no-console
+      console.log(JSON.stringify(body, null, 2))
```
Comment (hiagolcm) [inline node/clients/AdServer.ts:41]: I think adding the endpoint we are calling (`/api/ads/v1/ads/banner/query`) would help too.

File: node/resolvers/sponsoredBanners.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import AdServer from '../clients/AdServer'
+import type { AdServerSponsoredBannersResponse } from '../typings/AdServer'
+
+const AD_TYPE = 'banner'
+const SPONSORED_COUNT = 1
+
+const mapSponsoredBanner = (
```
Comment (hiagolcm) [inline node/resolvers/sponsoredBanners.ts:7]: I think `mapSponsoredBanner` doesn't do justice to what the function does. You could "map" a function for many reasons. Even though it describes the method you used, it doesn't describe what the function in fact does.

**Review by hiagolcm — CHANGES_REQUESTED** [review]

File: node/clients/AdServer.ts (hunk 1 above)
Comment (luarakerlen) [reply node/clients/AdServer.ts:41]: You mean to print the endpoint too?

Comment (hiagolcm) [reply node/clients/AdServer.ts:41]: yep!
Now there are two different endpoints. so It would be important to be able do tell the difference between then

**Review by hiagolcm — APPROVED** [review]

**Pull request merged**